test/unit/vm/json_schema/validation_test.dart

```diff
@@ -226,6 +226,79 @@ void main([List<String> args]) {
     // equivalent to the JSON input.
     'integer : a bignum is an integer',
     'integer : a negative bignum is an integer',
+    // Skip new tests from the spec that we don't pass yet:
+    'validation of e-mail addresses : ignores integers',
```

Comment:
These are new tests that are failing and need to be skipped. This is the only actual code change in the PR.